### PR TITLE
add roll dXX syntax

### PR DESCRIPTION
**Pull request by mihrits**
!roll d20 and so on...

### CODE REVIEW

File: rollbot/roller.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    logger.info(f"Rolling {rollCommand}")
+    modifier = 0
+    dice = rollCommand
+    if '+' in rollCommand:
+      dice, modifier = rollCommand.split('+')
+      modifier = int(modifier)
+    if '-' in rollCommand:
+      dice, modifier = rollCommand.split('-')
+      modifier = -int(modifier)
+    if re.fullmatch('d\\d+', dice):
```
Comment (hannilo) [inline rollbot/roller.py:24]: Could just test `dice.startswith('d')`

File: .env [hunk 2]
```diff
@@ -1,3 +1,3 @@
-TOKEN=NzA3MjUzNDM4NTM4NTc5OTk5.XrGHQg.b6Wbc3CblHl3UhEZghsz4-ovCzg
+TOKEN=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (hannilo) [inline .env]: Rebase and squash to purge history

**Review by hannilo — CHANGES_REQUESTED** [review]
Rebase

File: .env (hunk 2 above)
Comment (mihrits) [reply .env]: Discord nähtavasti skännib netti ja kohe kui ma pushisin nad resettisid mu tokeni ära.

File: rollbot/CommandManager.py [hunk 3]
```diff
@@ -64,14 +64,15 @@ def handle(self, message: discord.Message) -> CommandResult:
 
     if botCommand == Command.ROLL:
       if not args:
-        return ReplyResult(messageContent, False, f"{message.author.mention}, mkm")
+        result = self.roller.roll('1d20')
```
Comment (hannilo) [inline rollbot/CommandManager.py:67]: Set args[0] here for later use instead of repeating these 2 lines

**Review by hannilo — CHANGES_REQUESTED** [review]

**Comment by hannilo** [comment]
Done elsewhere

**Pull request closed without merging**